src/test/kotlin/racing/RacingGameAppTest.kt

```diff
@@ -35,13 +33,60 @@ class RacingGameAppTest {
         "3, false",
         "5, true",
     )
-    fun `입력 값 이동 테스트`(
+    fun `입력 숫자에 대한 이동 여부 테스트`(
         input: Int,
         expectedMoved: Boolean,
     ) {
         val initPosition = 5
-        val carModel = CarModel(input.toString(), TestController(input), initPosition)
+        val carModel =
+            CarModel(
+                input.toString(),
+                { input },
+                initPosition,
+            )
         carModel.nextState()
         (initPosition != carModel.currentPosition) shouldBe expectedMoved
     }
+
+    @ParameterizedTest
+    @ValueSource(ints = [2, 3, 4, 5])
+    fun `CarModel 위치 생성자 테스트`(input: Int) {
+        val carModel =
+            CarModel(
+                input.toString(),
+                controller = RacingController(),
+                initPosition = input,
+            )
+        carModel.currentPosition shouldBe input
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = ["1", "22", "333", "4444", "55555"])
+    fun `CarModel 이름 생성자 성공 테스트`(input: String) {
+        // - 각 자동차에 이름을 부여할 수 있다. 자동차 이름은 5자를 초과할 수 없다.
```

Comment:
테스트명 자체를 이렇게 설정하시면 시나리오가 더욱 명시적으로 변경됩니다!
```suggestion
    fun `각 자동차에 이름을 부여할 수 있다. 자동차 이름은 5자를 초과할 수 없다`(input: String) {
```

Reply:
이러한 시나리오는 `RacingGameAppTest.kt`에 두시기보다, `CarModel.kt`를 따로 만들어보시면 어떨까요?